src/scry.cr

```diff
@@ -39,6 +39,15 @@ module Scry
   ensure
     Log.logger.close unless Log.logger.nil?
   end
+
+  def self.process_error(request : Protocol::RequestMessage, error)
+    Protocol::ResponseMessage.new(request.id, error)
+  end
+
+  def self.process_error(request : Protocol::NotificationMessage, error)
+    Log.logger.error(%(#{error.message}\n#{error.backtrace.join('\n')}))
```

Comment:
It's good to log the error here, but can you also log an error when the request is a `RequestMessage`?